webservice/DMRService.py

```diff
@@ -1263,7 +1283,7 @@ def create_recovery_dataset(self, submission: Submission, rtype, rlist=None, tes
             
             if nfiles == 0:
                 logit.log("DMR-Service  | create_recovery_dataset | no matching files exist | recovery not needed")
-                return 0, None, None
+                return 0, None, None, []
```

Comment:
Does the final "return" (line 1345/1365) also need updating to " return 0, None, None, []"

Reply:
Yes!  Thanks.  I'll get that added.